src/plugins/ui_feature/insert_pivot.ts

```diff
@@ -99,7 +101,7 @@ export class InsertPivotPlugin extends UIPlugin {
       name = `${pivotName} (${i})`;
       i++;
     }
-    return name;
+    return name.replace(new RegExp(FORBIDDEN_IN_EXCEL_REGEX, "g"), " ");
```

Comment:
```suggestion
    let name = pivotName.replace(new RegExp(FORBIDDEN_IN_EXCEL_REGEX, "g"), " ");
    while (names.includes(name)) {
      name = `${pivotName} (${i})`;
      i++;
    }
    return name;
```

We could have a collision if the replacement is done after the check of existing sheet

Reply:
Good catch !

Fixed and test added :)